src/test/groovy/com/intuit/graphql/orchestrator/integration/federation/EntityTypeSpec.groovy

```diff
@@ -68,58 +63,27 @@ class StitchingTopLevelSpec extends BaseIntegrationTestSpecification {
         queryType.getFieldDefinition("getStoreByIdAndName") != null
     }
 
-    def "Federation shared value types do not conflict"(){
-        given:
-        def schema1 = """
-                    type Query {
-                        getProvider1Val: sharedValueType
-                    }
-                    type sharedValueType {
-                        id: ID!
-                        name: String
-                    }
-                """
-
-        def schema2 = """
-                    type Query {
-                        getProvider2Val: sharedValueType
-                    }
-                    type sharedValueType {
-                        id: ID!
-                        name: String
-                        test: String
-                    }
-                """
-
-        def valueProvider1 = TestServiceProvider.newBuilder()
-                .namespace("A")
-                .serviceType(ServiceProvider.ServiceType.FEDERATION_SUBGRAPH)
-                .sdlFiles(ImmutableMap.of("schema1", schema1))
-                .build()
-
-        def valueProvider2 = TestServiceProvider.newBuilder()
-                .namespace("B")
-                .serviceType(ServiceProvider.ServiceType.FEDERATION_SUBGRAPH)
-                .sdlFiles(ImmutableMap.of("schema2", schema2))
-                .build()
-
+    def "Federation entities can be extended with extends directive and extend keyword"() {
```

Comment:
Should we keep a test that fails when shared values do not use `extend` keyword?